readme.md

```diff
@@ -160,16 +160,15 @@ assumes you've installed Lighthouse as a dependency (`yarn add --dev lighthouse`
 
 ```javascript
 const lighthouse = require('lighthouse');
-const ChromeLauncher = require('lighthouse/chrome-laucher/chrome-launcher.js');
+const ChromeLauncher = require('lighthouse/chrome-launcher/chrome-launcher.js');
 const Printer = require('lighthouse/lighthouse-cli/printer');
 
 function launchChromeAndRunLighthouse(url, flags, config) {
   const launchedChrome = ChromeLauncher.launch();
 
   launchedChrome
     .then(() => lighthouse(url, flags, config)) // Run Lighthouse.
-    .then(results => launchedChrome.kill())
-    .then(() => results) // Kill Chrome and return results.
+    .then(results => launchedChrome.kill().then(() => results)) // Kill Chrome and return results.
```

Comment:
`results` isn't in scope. You could stash a var, but I actually think that complicates the example.